virt/virt-4-13-release-notes.adoc

```diff
@@ -123,20 +125,21 @@ All VM templates that are included with {VirtProductName} now use this machine t
 //CNV-25541 Release note: NEW -- UI can disable graphical display
 * You can now enable headless mode for high performance VMs in the web console.
 
-//NOTE: Comment out deprecated and removed features (and their IDs) if not used in a release
-//[id="virt-4-13-deprecated-removed"]
-//== Deprecated and removed features
-
 
-//[id="virt-4-13-deprecated"]
-//=== Deprecated features
+[id="virt-4-13-deprecated-removed"]
+== Deprecated and removed features
 // NOTE: when uncommenting deprecated features list, change the header level below to ===
 
-//Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
+[id="virt-4-13-deprecated"]
+=== Deprecated features
 
+Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
+
+//CNV-27160 
+* In a future release, support for `virtctl` command line tool installation for {op-system-base-full} 7 and {op-system-base-full} 9 by RPM will be removed.
```

Comment:
Though it might be clear that the `virtctl` command line tool will be removed in a future release, it is not clear that the `virtctl` command line tool is deprecated in the current release. Consider re-wording for clarity. For more information, see [Deprecated functionality](https://redhat-documentation.github.io/supplementary-style-guide/#_deprecated_functionality) and [Deprecation notice](https://redhat-documentation.github.io/supplementary-style-guide/#_deprecation_notice).